addons/portal_rating/static/src/xml/portal_tools.xml

```diff
@@ -36,32 +36,33 @@
             <div t-attf-class="#{two_columns and 'col-lg-12' or 'col-lg-6'}" t-if="!_.isEmpty(widget.get('rating_card_values'))">
                 <hr t-if="two_columns"/>
                 <p t-if="!two_columns"><strong>Details</strong></p>
-                <div class="o_website_rating_progress_bars">
-                    <table class="o_website_rating_progress_table">
-                        <t t-foreach="widget.get('rating_card_values')['percent']" t-as="percent">
-                            <tr class="o_website_rating_select" t-att-data-star="percent['num']" style="opacity: 1">
-                                <td class="o_website_rating_table_star_num text-nowrap" t-att-data-star="percent['num']">
-                                    <t t-esc="percent['num']"/> stars
-                                </td>
-                                <td class="o_website_rating_table_progress">
-                                    <div class="progress">
-                                        <div class="progress-bar o_rating_progressbar" role="progressbar" t-att-aria-valuenow="percent['percent']" aria-valuemin="0" aria-valuemax="100" t-att-style="'width:' + percent['percent'] + '%;'">
-                                        </div>
+                <t t-set="selected_rating" t-value="widget.get('rating_value')"/>
+                <table t-attf-class="o_website_rating_table #{selected_rating and 'o_website_rating_table_has_selection' or ''}">
+                    <t t-foreach="widget.get('rating_card_values')['percent']" t-as="percent">
+                        <t t-set="row_selected" t-value="percent['num'] == selected_rating"/>
+                        <tr t-attf-class="o_website_rating_table_row #{selected_rating and !row_selected and 'o_darken_5' or ''} #{!row_selected and 'o_pointer' or ''}"
```

Comment:
`o_darken_5` can be replaced with bootstrap's `opacity-50`.
And to reduce tests to handle row selection, what do you think about this slightly less readable line ?
```python
!selected_rating and '' or row_selected and 'o_website_rating_table_row_selected' or 'opacity-50'
```
(See other comment for `o_website_rating_table_row_selected`)

Reply:
Thanks for the proposition. I must admit that I find those expressions hard to understand so I have a counter proposition which also group the 2 expression in one:
```javascript
     row_selected?'o_website_rating_table_row_selected':(selected_rating?'opacity-50':'')
```
It is a bit less pythonic but the former expression was also an hybrid due to the fact that "not" is not working and "!" is used instead. What do you think ?

Reply:
Great! (maybe some spaces between operators ?)

Reply:
Yes, I have used js formatting